README.md

```
@@ -1,73 +1,271 @@
+
+---
+
+## 👥 팀 소개
+
+> 공공데이터 활용 아이디어 공모전 2026 출품작
+
+<div align="center">
+
+| **이세비** | **조지현** | **박지현** |
+|:---:|:---:|:---:|
+| <img src="https://github.com/2seb2.png" width="80" style="border-radius:50%" /> | <img src="https://github.com/zeon0xx0.png" width="80" style="border-radius:50%" /> | <img src="https://github.com/jjhparkk.png" width="80" style="border-radius:50%" /> |
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)